python_modules/dagster/dagster/_config/pythonic_config/__init__.py

```diff
@@ -1699,3 +1741,18 @@ def validate_resource_annotated_function(fn) -> None:
                 else "'ResourceParam[Any]' or 'ResourceParam[<output type>]'",
             )
         )
+
+
+class _FixedConfig:
+    def __init__(self, value: Any):
+        self._value = value
+
+    def get_value(self) -> Any:
+        return self._value
+
+
+C = TypeVar("C")
+
+
+def FixedConfig(value: C) -> C:
```

Comment:
This is pretty hacky and I'm pretty hesitant to ship something like this. There may be a better way to handle this (or just return `Any`) that will not cause type issues.